sdk/network/mgmt/src/main/java/com/azure/management/network/implementation/NetworkSecurityRuleImpl.java

```diff
@@ -162,7 +162,7 @@ public NetworkSecurityRuleImpl withProtocol(SecurityRuleProtocol protocol) {
 
     @Override
     public NetworkSecurityRuleImpl withAnyProtocol() {
-        return this.withProtocol(SecurityRuleProtocol._);
+        return this.withProtocol(SecurityRuleProtocol.fromString("*"));
```

Comment:
Better still use the enum

Reply:
I think from string will not change at all. Instead, the enum name is still not ready after discussed.

Reply:
The point is when generator changes it, no compiler will tell anyone anything about this line.

Reply:
Yes, I change it back. But I think the `any` should not be other char than '*'.